src/forwarder.ts

```diff
@@ -28,12 +39,25 @@ interface DescribeSubscriptionFiltersResponse {
 function isLogGroup(value: any): value is LogGroupResource {
   return value.Type === logGroupKey;
 }
+/**
+ * Validates whether ARN exists in the account
+ * @param aws Serverless framework provided AWS client
+ * @param functionArn The ARN to be validated
+ */
+async function doesArnExist(aws: Aws, functionArn: string) {
```

Comment:
I would rename this function to `doesFunctionExist` -- ARNs are used for lots of things besides functions, and this code specifically looks for a Lambda function.